### DISCUSSION TITLE
Wordpress website sharing with Ngrok

### DISCUSSION THREAD

**Discussion by Amos-Sang** [leokhoa/laragon#781] [General]
I am trying to share my local website within the LAN using Ngrok, it seems the tunnel is established but Laragon returns the 
error "could not detect tunnel link". And also, the link established loads the default laragon page instead of the website.
![Screenshot 2024-05-14 140031](https://github.com/leokhoa/laragon/assets/123357827/25483c52-3f66-4ef5-bbc5-32d24e267ea2)
![Screenshot 2024-05-14 140118](https://github.com/leokhoa/laragon/assets/123357827/eab0af0a-3b72-4a15-b5d0-391103f48ae1)

**Comment by josealgardataborges** [comment]
Was getting some erros on LARAGON / NGROK, similar to yours (i got the same laragon error message, but no ngrok link).

ERROR:  authentication failed: Your account is limited to 1 simultaneous ngrok agent sessions.
ERROR:  You can run multiple tunnels on a single agent session using a configuration file.
ERROR:  To learn more, see https://ngrok.com/docs/secure-tunnels/ngrok-agent/reference/config/
ERROR:  You can view your current agent sessions in the dashboard:
ERROR:  https://dashboard.ngrok.com/tunnels/agents
ERROR:
ERROR:  ERR_NGROK_108
ERROR:


For me the only solution that worked was going to ngrok dashboard, and kill the session that was there, the free plan only allows 1 session at a time


Go to this page **https://dashboard.ngrok.com/tunnels/agents** click the **STOP** button has seen bellow and choose **STOP SESSION**

![image](https://github.com/leokhoa/laragon/assets/8929655/975413b1-e1fb-4166-b7d6-e054af45de92)

![image](https://github.com/leokhoa/laragon/assets/8929655/cff162b4-8c14-4b36-8661-f8b235883b06)

Then return to **LARAGON** and try again :) it should work now.
